Add maximum allowed monitor number. (#3732)
opensearch-project/documentation-website: _observing-your-data/alerting/monitors.md

1. Specify a name for the monitor.
1. Choose either **Per query monitor**, **Per bucket monitor**, **Per cluster metrics monitor**, or **Per document monitor**.

> Per query monitors run your specified query and then check whether the query's results trigger any alerts. Per bucket monitors let you select which fields to create buckets and categorize your results into those buckets. The Alerting plugin runs each bucket's unique results against a script you define later, so you have finer control over which results should trigger alerts. Each of those buckets can trigger an alert, but query-level monitors can only trigger one alert at a time. The maximum number of monitors you can create is 1,000 and is configurable if you call the cluster settings API `plugins.alerting.monitor.max_monitors` to change the settings for the maximum monitor count.

Reviewer: ```suggestion
Per query monitors run your specified query and then check whether the query's results trigger any alerts. Per bucket monitors let you select which fields to create buckets and categorize your results into those buckets. The Alerting plugin runs each bucket's unique results against a script you define later, so you have finer control over which results should trigger alerts. Each bucket can trigger an alert, but the query-level monitors can only trigger one alert at a time. The maximum number of monitors you can create is 1,000 and is configurable if you call the cluster settings API `plugins.alerting.monitor.max_monitors` to change the settings for the maximum monitor count.
```
```suggestion
Per query monitors run your specified query and then check whether the query's results trigger any alerts. Per bucket monitors let you select which fields to create buckets and then categorize your results into those buckets. 

The Alerting plugin runs each bucket's unique results against a script you define later, so you have finer control over which results should trigger alerts. Each bucket can trigger an alert, but query-level monitors can only trigger one alert at a time. The maximum number of monitors you can create is 1,000 and is configurable if you call the cluster settings API `plugins.alerting.monitor.max_monitors` to change the settings for the maximum monitor count.
```

Reviewer 2: I would reformat this section for readability, using the wording changes proposed by Melissa:

```suggestion

OpenSearch supports the following types of monitors:

- **Per query monitors** run your specified query and then check whether the query's results trigger any alerts. Query-level monitors can only trigger one alert at a time.
- **Per bucket monitors** let you select which fields to create buckets and categorize your results into those buckets.  The Alerting plugin runs each bucket's unique results against a script you define later, so you have finer control over which results should trigger alerts. Furthermore, each bucket can trigger an alert.

The maximum number of monitors you can create is **1,000**. If you want to change the default maximum number of alerts for your cluster to below 1000, you can is configure the maximum by calling the cluster settings API `plugins.alerting.monitor.max_monitors`.
```

<!--- You can only configure the maximum number of monitors in open source OpenSearch.
{: .note}---> 

> 1. Decide how you want to define your query and triggers. You can use any of the following methods: visual editor, query editor, or anomaly detector.

Reviewer 2: We describe the methods in the section below, though we could stand to readdress the formatting and flow of this page generally speaking. I can make an issue for that.

1. Specify a name for the monitor.
1. Choose either **Per query monitor**, **Per bucket monitor**, **Per cluster metrics monitor**, or **Per document monitor**.

OpenSearch supports the following types of monitors:

> - **Per query monitors** run your specified query and then check whether the query's results trigger any alerts. Query-level monitors can only trigger one alert at a time. 

Reviewer 3: Should "Query-level monitors" be "Per query monitors"?

Author: This should be "Per query monitors".

1. Specify a name for the monitor.
1. Choose either **Per query monitor**, **Per bucket monitor**, **Per cluster metrics monitor**, or **Per document monitor**.

OpenSearch supports the following types of monitors:

> **Per query monitors** run your specified query and then check whether the query's results trigger any alerts. Query-level monitors can only trigger one alert at a time. 

Author: Bulleting was suggested in doc review, but I don't see the content requiring that to make the content clear.

> **Per query monitors** run your specified query and then check whether the query's results trigger any alerts. Query-level monitors can only trigger one alert at a time. 

Reviewer 3: If we're providing an unordered list of items or terms, as we are here, then they should be formatted as a bulleted list.

> **Per query monitors** run your specified query and then check whether the query's results trigger any alerts. Query-level monitors can only trigger one alert at a time. 

Author: I misinterpreted the ask here - I'll make sure these items are bulleted. Thanks!

> **Per bucket monitors** let you select which fields to create buckets and categorize your results into those buckets. The Alerting plugin runs each bucket's unique results against a script you define later, so you have finer control over which results should trigger alerts. Furthermore, each bucket can trigger an alert.

Reviewer 3: The first sentence needs a little revision. Do we mean something like "**Per bucket monitors** let you create buckets based on selected fields and then categorize your results into those buckets"?

> The maximum number of monitors you can create is **1,000**. If you want to change the default maximum number of alerts for your cluster to below **1,000**, you can configure the maximum number of alerts by calling the cluster settings API `plugins.alerting.monitor.max_monitors`.

Author: @natebower Does the max number of monitors (1,000) count as a constraint? In that case, it shouldn't be bold, right?

Reviewer 3: It should be bold if it appears as a UI element, if our formatting guidelines indicate that it should be, or if we particularly want to emphasize it. Otherwise, no.

> The maximum number of monitors you can create is **1,000**. If you want to change the default maximum number of alerts for your cluster to below **1,000**, you can configure the maximum number of alerts by calling the cluster settings API `plugins.alerting.monitor.max_monitors`.

Reviewer 3: ```suggestion
The maximum number of monitors you can create is **1,000**. You can change the default maximum number of alerts for your cluster by calling the cluster settings API `plugins.alerting.monitor.max_monitors`.
```

> <!--- You can only configure the maximum number of monitors in open source OpenSearch.

Reviewer 3: Will this appear in the doc? If so, it needs revision.

Author: This will not be included.
